synths/core-synths.scd

```diff
@@ -314,6 +314,30 @@ live coding them requires that you have your SuperDirt instance in an environmen
 
 		ReplaceOut.ar(out, signal);
 	}, [\ir, \ir, \ir]).add;
+	
+	// Phaser audio DSP effect definition. Params: phaserrate (Hz); phaserdepth (amplitude)
+	SynthDef("dirt_phaser" ++ numChannels, { |out, phaserrate = 1.0, phaserdepth = 0.5 |
+		var signal, phaserwave;
+		var maxDelayTime = 4;
+		var phase =  LFPar.kr(phaserrate).range(0.0088, 0.01);
+
+		~phaserBus = Bus.audio(s, 2);
```

Comment:
No need to test, I know it from reading the code. Unless you specifically prevent this, two instances of SuperDirt will run in the same environment. Using dynamic scope (`~something`) you will override them. Normally, unless in spontaneous hacks, you don't really use such constructions in a `SynthDef`.